### PR TITLE
Feature/tabs

### PR DESCRIPTION
**Pull request by fraincs**
- Rebrand of Tabs

IGL-67

### CODE REVIEW

File: packages/Tabs/src/tabs.scss [hunk 1]
```diff
@@ -1,115 +1,212 @@
...
 }
 
-.ids-tabs {
-  font-family: var(--ids-tabs-font-family);
-  font-size: var(--ids-tabs-font-size);
+[data-brand="workleap"] {
+    /* Default */
+    --ids-tabs-font-family: var(--hop-body-sm-font-family);
+    --ids-tabs-font-size: var(--hop-body-sm-font-size);
+    --ids-tabs-margin: #{tokens.$space-0} #{tokens.$space-0} 1.25rem;
```
Comment (franckgaudin) [inline packages/Tabs/src/tabs.scss:58]: Igloo tokens must be removed even if there are no equivalent tokens in 🐸Hopper.
```suggestion
    --ids-tabs-margin: 0 0 1.25rem;
```

File: packages/Tabs/src/tabs.scss [hunk 2]
```diff
@@ -1,115 +1,212 @@
...
+    --ids-inline-tab-outline: 0.0625rem solid var(--hop-neutral-border-weak);
+    --ids-inline-tab-outline-offset: -0.0625rem;
+    --ids-inline-tab-outline-focus-offset: -6px;
+    --ids-inline-tab-button-font-size: var(--hop-body-sm-font-size);
+    --ids-inline-tab-button-line-height: var(--hop-body-sm-line-height);
+    --ids-inline-tab-button-focus-height: calc(100% - 1.5rem);
+    --ids-inline-tab-button-focus-width: calc(100% - 1.5rem);
+
+    /* Tab */
+    --ids-inline-tab-button-padding: calc(1rem - 0.125rem) 1rem;
```
Comment (franckgaudin) [inline packages/Tabs/src/tabs.scss:96]: Take Hopper's tokens here

File: packages/Tabs/src/tabs.scss [hunk 3]
```diff
@@ -1,115 +1,212 @@
...
+    font-family: var(--ids-tabs-font-family);
+    font-size: var(--ids-tabs-font-size);
+
+    &__list {
+        display: flex;
+        list-style: none;
+        margin: var(--ids-tabs-margin);
+        padding: 0;
+
+        .ids-tabs--inline &  {
```
Comment (franckgaudin) [inline packages/Tabs/src/tabs.scss:112]: There's a statement below about inline rendering, so why duplicate it here?

Comment (fraincs) [reply packages/Tabs/src/tabs.scss:112]: notice the & at the end? This declaration is only for links/buttons that are inline where below the declaration is for inline tabs not the buttons inside an inline tab

**Review by alexasselin008 — APPROVED** [review]

**Review by franckgaudin — APPROVED** [review]

**Pull request merged**